### PR TITLE
Fix/dangerslugs 809

### PR DESCRIPTION
**Pull request by soulseekah**

### CODE REVIEW

**Comment by soulseekah** [comment]
#809

**Comment by soulseekah** [comment]
This is done.

For @zackkatz:

- [ ] check and modify language strings as needed
- [ ] check and publish documentation
- [ ] set documentation URL in the warning message
- [ ] PR and merge

whew.. this was a tough one

**Comment by zackkatz** [comment]
Nice job, @soulseekah !

File: future/includes/class-gv-renderer.php [hunk 1]
```diff
@@ -16,8 +16,8 @@ class Renderer {
 	 * Initialization.
 	 */
 	public function __construct() {
-		if ( ! has_action( 'gravityview/template/after', array( __CLASS__, 'maybe_print_notices' ) ) ) {
-			add_action( 'gravityview/template/after', array( __CLASS__, 'maybe_print_notices' ) );
+		if ( ! has_action( 'gravityview/template/before', array( __CLASS__, 'maybe_print_notices' ) ) ) {
```
Comment (soulseekah) [inline future/includes/class-gv-renderer.php]: We have to keep in mind that this also moves the unconfigured warning to the top of the view. Might have some (non-immediate?) implications.

Comment (zackkatz) [reply future/includes/class-gv-renderer.php]: Yeah I tested it and I like it better than below. Not worried about it.

**Pull request merged**